### PR TITLE
OADP-2493 Added a section for info about disable/enable node agent

### PR DESCRIPTION
**Pull request by shdeshpa07**
## Jira 

* [OADP-2493](https://issues.redhat.com/browse/OADP-2493)

Added a section for info about disable/enable node agent

##  Version

* OCP 4.13 → OCP 4.17

## Preview

* [AWS - About disabling node agent in the DataProtectionApplication](https://79683--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.html#oadp-about-disable-node-agent-dpa_installing-oadp-aws)
* [Azure - About disabling node agent in the DataProtectionApplication](https://79683--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-azure.html#oadp-about-disable-node-agent-dpa_installing-oadp-azure)
* [GCP - About disabling node agent in the DataProtectionApplication](https://79683--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp.html#oadp-about-disable-node-agent-dpa_installing-oadp-gcp)
* [MCG - About disabling node agent in the DataProtectionApplication](https://79683--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.html#oadp-about-disable-node-agent-dpa_installing-oadp-mcg)
* [ODF - About disabling node agent in the DataProtectionApplication](https://79683--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.html#oadp-about-disable-node-agent-dpa_installing-oadp-ocs)

## QE Review

* [x] QE has approved this change.

### CODE REVIEW

**Comment by shdeshpa07** [comment]
@anarnold97 - Could you please review this PR?

**Comment by shdeshpa07** [comment]
@PrasadJoshi12 - Could you please review this PR? Thanks.

**Review by PrasadJoshi12 — APPROVED** [review]
LGTM

**Review by lpettyjo — COMMENTED** [review]
You have two commits. Please squash to a single commit. For help, see https://docs.google.com/document/d/1fLMpK4bqthtFlCwA36yeo2cI-pExFpX76VMnSi_In6Q/edit#heading=h.epes4xf06pzk.

Otherwise, LGTM!

File: modules/oadp-about-disable-node-agent-dpa.adoc [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oadp-about-disable-node-agent-dpa_{context}"]
+= About disabling node agent in the DataProtectionApplication
+include::_attributes/common-attributes.adoc[]
+
+If you are not using `Restic`, `Kopia`, or `DataMover` for your backups, you can consider disabling the `nodeAgent` in the `DataProtectionApplication` as shown below. Make sure the {oadp-short} Operator is idle and not running any backups, before you disable the `nodeAgent`.
```
[the remark this answers is not in the document]
Comment (shdeshpa07) [reply modules/oadp-about-disable-node-agent-dpa.adoc]: Incorporated the change

File: modules/oadp-about-disable-node-agent-dpa.adoc [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+# ...
+configuration:
+  nodeAgent:
+    enable: false  # <1>
+    uploaderType: kopia
+# ...
+----
+<1> Disable node agent.
+
+To enable the `nodeAgent`, set the `enable` flag to `true` as shown below:
```
[the remark this answers is not in the document]
Comment (shdeshpa07) [reply modules/oadp-about-disable-node-agent-dpa.adoc]: Incorporated feedback.

File: modules/oadp-about-disable-node-agent-dpa.adoc [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-azure.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oadp-about-disable-node-agent-dpa_{context}"]
+= About disabling node agent in the DataProtectionApplication
+include::_attributes/common-attributes.adoc[]
```
[the remark this answers is not in the document]
Comment (kcarmichael08) [reply modules/oadp-about-disable-node-agent-dpa.adoc:12]: Hi, I notice that you resolved this issue, but it is actually causing the cherry picks to fail, I believe. It looks like there are no includes in modules for the common attributes, according to the guidelines:

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

I think this needs to be removed:

include::_attributes/common-attributes.adoc[]

**Review by kcarmichael08 — COMMENTED** [review]
This is failing because there is an include for the common attributes file in a module.

**Comment by kcarmichael08** [comment]
Hi @shdeshpa07 , can you please resubmit this as a new PR, and omit the include for the common attributes file in the module in that new PR? Please link back to this PR in the body of the PR. I closed all the cherry picks and submitted a PR to reverse this PR merge, because it will cause problems having the include for that file. Thank you.

**Pull request merged**